routers/router.sin1.yml

```
@@ -79,3 +79,13 @@
     remote_address: sin05.dn42.ventilaar.net
     remote_port: 30207
     public_key: h6h4yzNS/jMPvrNzD+er1zoAPcp7pOlPuWtnemrOZxQ=
+
+- name: CMCC-HK
+  asn: 4242421686
+  ipv6: fe80::1686
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
+    public_key: ocG/0wXtmuXA9OF5pMVolBIsFfE1K3JILX3MC3J5pGI=
```

```suggestion
    public_key: ocG/0wXtmuXA9OF5pMVolBIsFfE1K3JILX3MC3J5pGI=

```

Add blank line to end of file.